clusterloader2/testing/load/config.yaml

```diff
@@ -135,184 +133,29 @@ steps:
       - basename: daemonset-priorityclass
         objectTemplatePath: daemonset-priorityclass.yaml
```

Comment:
I don't have strong opinion, but maybe we could add this and deleting of priority class to new module. Creating daemonsets depends on it so I think it makes sense. @mborsz do you have any opinion about it?

Reply:
@mborsz

Reply:
Let's keep it here for now.

Preferably I would make priorityClassName to a module's parameter and pass 'daemonset-priorityclass-0' there.
This way it will be the module caller's responsiblity to pass somewhere there and it will be clear what is a relation between daemonset-priorityclass and daemonset objects.